frappe/utils/__init__.py

```diff
@@ -528,13 +528,6 @@ def get_request_site_address(full_address=False):
 	return get_url(full_address=full_address)
 
 
-def get_site_url(site):
```

Comment:
Instead of removing completely lets add a alias for this. 

It's used in at least one other project: https://sourcegraph.com/search?q=context:global+get_site_url+repo:%5Egithub%5C.com/frappe/.*&patternType=standard&sm=0&groupBy=repo